scripts/make_gitrev.sh

```diff
@@ -39,3 +39,7 @@ trap cleanup EXIT
 git rev-parse --git-dir > /dev/null 2>&1
 # Get the git current git revision
 ZFS_GIT_REV=$(git describe --always --long --dirty 2>/dev/null)
+# Check if header file already contain the exact string
+grep -sq "\"${ZFS_GIT_REV}\"" "$(dirname "$0")"/../include/zfs_gitrev.h &&
+	trap - EXIT
+exit 0
```

Comment:
The exit is just here when grep returns non-zero returncode. Ie, if the file does not exist. I had `|| true` for a while too, not sure which is preferred. It is just to stop `make` from complaining about the script "failing" - so it more or less should always return 0. Thoughts?

Reply:
Adding the `exit 0` makes more sense to me.